src/modules/dashboard/components/ExpendituresDialog/ExpendituresDialog.tsx

```diff
@@ -73,26 +73,25 @@ const ExpendituresDialog = ({
   prevStep,
   colony: { colonyAddress },
   colony,
+  isVotingExtensionEnabled,
   nextStep,
 }: Props) => {
   const { walletAddress, username, ethereal } = useLoggedInUser();
+  const { isOneTxPaymentExtensionEnabled } = useEnabledExtensions({
+    colonyAddress,
+  });
+
+  // const isVotingExtensionEnabled = false;
```

Comment:
I'm assuming this is a leftover debug value

Reply:
of course :)